.jshintrc

```diff
@@ -2,5 +2,7 @@
     "curly": true,
     "eqeqeq": true,
     "undef": true,
-    "esnext": true
+    "esnext": true,
+    "devel": true,
```

Comment:
I would remove it. This way we'll see at build time if we have some debug in our files.